### PR TITLE
FHIR Model Guide

### PR DESCRIPTION
**Pull request by JohnTimm**

### CODE REVIEW

File: docs/FHIRModelGuide.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Several static factory / utility methods are also used:
+
+- `Canonical.of(...)`
+- `Uri.of(...)`
+- `Code.of(...)`
+- `String.string(...)` (via static import)
+
+Many of the primitive data types contain this type of "helper" method.
+
+Fields from an immutable model object may be copied back into a builder object using the `toBuild()` method:
```
Comment (lmsurpre) [inline docs/FHIRModelGuide.md]: toBuilder()

File: fhir-validation/pom.xml [hunk 2]
```diff
@@ -23,6 +23,11 @@
             <artifactId>fhir-registry</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-ig-us-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```
Comment (lmsurpre) [inline fhir-validation/pom.xml]: why does fhir-validator depend on a specific ig?

File: docs/FHIRModelGuide.md (hunk 1 above)
Comment (JohnTimm) [reply docs/FHIRModelGuide.md]: Fixed

File: fhir-validation/pom.xml (hunk 2 above)
Comment (JohnTimm) [reply fhir-validation/pom.xml]: Fixed

**Pull request merged**